pkg/cmd/roachprod/main.go

```diff
@@ -860,11 +860,13 @@ var stageURLCmd = &cobra.Command{
 	Long: `Prints URL for release and edge binaries.
 
 Currently available application options are:
-  cockroach - Cockroach Unofficial. Can provide an optional SHA, otherwise
-              latest build version is used.
-  workload  - Cockroach workload application.
-  release   - Official CockroachDB Release. Must provide a specific release
-              version.
+  cockroach  - Cockroach Unofficial. Can provide an optional SHA, otherwise
+               latest build version is used.
+  workload   - Cockroach workload application.
+  release    - Official CockroachDB Release. Must provide a specific release
+               version.
+  customized - Customized CockroachDB build. Must provide a specific build
```

Comment:
How is customized build different from "Cockroach Unofficial"? It's not immediately clear from the description.

Reply:
Good question. Cockroach Unofficial is built using merged source code (aka edge binaries). Customized builds are built using using `./scripts/tag-custom-build.sh` and the source code is not merged to the main repo (usually). They live in different buckets as well.

Reply:
I can update the comments for both types.